src/librustc/lint/builtin.rs

```diff
@@ -273,6 +274,12 @@ declare_lint! {
     "detects name collision with an existing but unstable method"
 }
 
+declare_lint! {
+    pub DUPLICATED_MACRO_EXPORTS,
```

Comment:
I've just realized that "duplicate" may be more correct English here than "duplicated".